File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freq={}
        for i in nums:
            if i not in freq:
                freq[i]=0
            freq[i]+=1
        sorts = sorted(list(freq.items()),key=lambda pair:pair[1],reverse=True)
        values = []
        counter =0
        for x in sorts:
            if counter!=k:
                values.append(x[0])
                counter+=1
        
        return values
        
        
          
        
       

File: Data Structures & Algorithms/duplicate-integer/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        for(let i =0; i<nums.length-1; i++){
            for(let k =i+1;k<nums.length;k++){
                if(nums[i] == nums[i+1]){
                    return true
            }
                return false
            }
        }
    }
}
